Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -14,7 +14,7 @@ class LivecheckCmd < AbstractCommand
           Check for newer versions of formulae and/or casks from upstream.
           If no formula or cask argument is passed, the list of formulae and
           casks to check is taken from `HOMEBREW_LIVECHECK_WATCHLIST` or
-          `~/.homebrew/livecheck_watchlist.txt`.
+          `~/.homebrew/livecheck_watchlist.txt`, excluding autobumped formulae.
```

Comment:
If we add a way to disable this filtering behavior, this statement may not always be true.

Reply:
```suggestion
          `~/.homebrew/livecheck_watchlist.txt`.
```